packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Okta Verify (Push/OTP) integration guide
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+<StackSelector />
+
+This guide shows you how to integrate Okta Verify into your app using the embedded SDK.
```

Comment:
This text is almost a repetition of the title... why is it here?

Reply:
Assuming it's an abstract of the article for SSO and for the developer to figure out if it's something they need, how about:

Integrate Okta Verify into your app using the embedded SDK.